packages/aws-cdk-lib/aws-cloudwatch/lib/metric.ts

```diff
@@ -303,6 +303,23 @@ export class Metric implements IMetric {
     });
   }
 
+  /**
+   * Make a new Alarm for this metric using anomaly detection
+   *
+   * @param scope The scope in which to create the alarm
+   * @param id The ID of the alarm
+   * @param props Alarm creation properties for anomaly detection
+   * @param metric The metric to create the alarm for
+   * @returns The newly created Alarm
+   */
+  public static createAnomalyDetectionAlarmFromMetric(scope: Construct, id: string, props: CreateAnomalyDetectionAlarmProps, metric: IMetric): Alarm {
```

Comment:
nit: I didn't find this meaningful when there's also `createAnomalyDetectionAlarm` method